.github/workflows/test.yaml

```diff
@@ -27,15 +27,22 @@ jobs:
         uses: ./.github/actions/dev-env-setup
       - run: yarn install --immutable
         working-directory: ./bciers
-  yarn-test:
+  nx-tests:
     needs: install-dev-tools
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./bciers
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
Do we have to go to v3? Marcel updated us to v4 awhile ago

Reply:
Thanks for the catch. I was moving a step from one place to anther that just happens to use the same action at a different version!